app/assets/stylesheets/gov.scss

```diff
@@ -17,10 +17,22 @@
 
 .govuk-grid-row .govuk-grid-column-full .govuk-grid-row .govuk-grid-column-one-half:nth-child(odd),
 .govuk-grid-row .govuk-grid-column-two-thirds .govuk-grid-row .govuk-grid-column-one-half:nth-child(odd) {
-  padding: 0 15px 0 0;
+  padding: 0;
 }
 
 .govuk-grid-row .govuk-grid-column-full .govuk-grid-row .govuk-grid-column-one-half:nth-child(even),
 .govuk-grid-row .govuk-grid-column-two-thirds .govuk-grid-row .govuk-grid-column-one-half:nth-child(even) {
-  padding: 0 0 0 15px;
+  padding: 0;
+}
+
+@media (min-width: 40.0625em) {
```

Comment:
feels like a weird breakpoint number - we might want to look at what the design-system uses for breakpoints

Reply:
That's true. I actually took this from inspecting the GovUK frontend CSS on the browser. I'll have a look deeper into the breakpoints it uses in the code instead.

Reply:
Updated in https://github.com/DFE-Digital/single-place-for-education-frontend/pull/167/commits/7aed41296d5c9dc4b78f38769d92a21aa653b5c7.